cpp/ql/src/Critical/FileMayNotBeClosed.ql

```diff
@@ -164,4 +164,4 @@ where
     fopenVariableReaches(v, def, ret) and
     ret.getAChild*() = v.getAnAccess()
   )
-select def, "The file opened here may not be closed at $@.", ret, "this exit point"
+select def, "This file open may not be closed at $@.", ret, "this exit point"
```

Comment:
I kind of prefer the old message here, I think it reads more easily.  Is this for consistency with another language?

Reply:
I'm trying to avoid having a non-descriptive "here" in the message.   
The query highlights `def`, and stating that something happened "here" is redundant.  

Although sometimes the version with "here" reads better. 

I might revert this (and `MemoryMayNotBeFreed.ql`).

Reply:
This one bothers me more than `MemoryMayNotBeFreed.ql`.  Calling the operation a "file open" feels a bit clunky.

Some alternatives:
 - "The file opened by this operation may not be closed at $@."
 - "This file may not be closed at $@."

Reply:
> "This file may not be closed at $@."

I like this one 👍